website/docs/docs/get-started/getting-started/overview.md

```diff
@@ -26,7 +26,7 @@ link="/docs/get-started/getting-started/set-up-dbt-cloud"
 
 <Card
     title="Getting started with dbt Core"
-    body="dbt Core is an open-source tool that enables data teams to transform data using analytics engineering best practices. "
+    body="When you use dbt Core to work with dbt, you will be editing files locally using a code editor, and running projects using a command line interface"
```

Comment:
might just be a Netlify thing but the card icons aren't rendering anymore: 

![Screen Shot 2022-10-12 at 3 13 21 PM](https://user-images.githubusercontent.com/107218380/195457892-69433f96-9cec-44f8-adf5-848159d90817.png)